### ISSUE TITLE
Search using previous visited sites instead of a new search

### ISSUE THREAD

**Issue by alexanderbarbosa** [atlas-engineer/next#256]
Hi,

How can I force Next to search for a term instead using recent visited pages?

Reproducing:
```
c-l
cnn 
Next list recent cnn visit history entries as: cnn.com
hit enter
go to cnn.com.
```
What I expect:
```
c-l
cnn
Next list history entries as: cnn.com
hit enter
https://duckduckgo.com/?q=cnn&ia=web
```
Same occurs with any search that have any term matching recent visited pages. 

Emacs' ido let users to ignore history with c-j and just do "what I mean"

I think that the default behavior should be always do what user mean, but if user want to use visited pages with another key or in a new way.

 **More**: https://stackoverflow.com/questions/5138110/emacs-create-new-file-with-ido-enabled

**Comment by Ambrevar** [comment]
Use `C-RETURN`.

This is somewhat unintuitive, and I believe a better way would be to do
like Helm does in Emacs, to propose the "raw input" as the first entry
of the list.

But was can't do that for all completion list (it does not always make
sense) and this needs some refactoring of the minibuffer completion system.

**Comment by Ambrevar** [comment]
This was also brought up in #206.  We need to work on this quite soon, I guess this confuses a lot of users.

**Comment by Ambrevar** [comment]
Fixed in a11666a0c28a41b8b529859423656a8068ebec94.
